contracts/FleekERC721.sol

```diff
@@ -6,10 +6,13 @@ import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
 import "@openzeppelin/contracts/utils/Counters.sol";
 import "@openzeppelin/contracts/utils/Base64.sol";
 import "./FleekAccessControl.sol";
+import "./util/FleekStrings.sol";
```

Comment:
Should we put these into a `libraries` folder?

Reply:
Here I'm following the structure from openzeppelin repo (https://github.com/OpenZeppelin/openzeppelin-contracts/tree/master/contracts/utils). But we can change it if you want, what do you think is best?

Reply:
I'm okay with this.